backend/inspirehep/search/mappings/v7/records/hep.json

```diff
@@ -1282,6 +1283,7 @@
             "initial_normalizer",
             "lowercase_normalizer"
           ],
+          "char_filter": ["dot_to_whitespace_normalizer"],
```

Comment:
does this run before or after the tokenizer? is it the right order to do what you want?

Reply:
👍 I'll read more about it, on my tests it worked as expected

Reply:
I run before the tokenizer
> Character filters are used to preprocess the stream of characters before it is passed to the tokenizer.

https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-charfilters.html